docs/docs.md

```diff
@@ -0,0 +1,585 @@
+# `/event/{eventKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  webcasts: {
+    type: "twitch" | "youtube"
+    url: string
+  }[]
+  location: {
+    name: string
+    lat: number
+    lon: number
+  }
+  name: string
+  district?: string
+  week?: number
+  // UTC date
+  startDate: string
+  // UTC date
+  endDate: string
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/info`
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  redAlliance: string[]
+  blueAlliance: string[]
+  // UTC date - match predicted time
+  time: string
+  // example: qm3
+  key: string
+}
+```
+
+# `/event/{eventKey}/match/{matchKey}/reports/{team}`
+
+## `PUT`
+
+### Request
+
+```ts
+type Data = {
+  teleop: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+  auto: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+}
+```
+
+
+### Response
+
+```ts
+type Data = null
+```
+
+
+## `GET`
+
+### Response
+
+```ts
+type Data = {
+  reporter: string
+  reporterId: string
+  teleop: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+  auto: (
+    | {
+        attempts: number
+        successes: number
+        statName: string
+      }
+    | {
+        attempted: boolean
+        succeeded: boolean
+        statName: string
+      })[]
+}[]
+```
+
+# `/event/{eventKey}/match/{matchKey}/stats`
```

Comment:
I feel like this needs a comment explaining that it is total stats for those teams not just stats for that match - I got tripped up when I first saw this. It's pretty vague what the scope here is - is it stats for the match? For the team, but only within the event? Or stats from all the data we have on that team at all events?

Reply:
:heavy_check_mark: